Articles/Blog/2025-08-BSSwFellowsOpen2025.md

```diff
@@ -0,0 +1,86 @@
+* Able to use the fellowship to broadly benefit the scientific software community.
+
+Applicants must be affiliated with a U.S.-based institution that is able to receive Federal funding.  Typically such institutions include U.S. government labs, U.S. universities, and U.S.-based corporations.
+
+### Q&A Webinar about the BSSw Fellowship Program
+
+Join us for a Q&A Webinar to hear more about the BSSw Fellowship Program and get your questions answered.
+- Tuesday, September 16, 2025, 1:00-2:00 PM EDT
+- Please [subscribe to our mailing list](https://bssw.io/pages/receive-our-email-digest) to receive further details, including telecon information for the Q&A session.
+
+### More information, including on-line application and FAQ
+
+Better Scientific Software is an organization dedicated to improving developer productivity and software sustainability for computational science and engineering through BETTER Planning, Development, Performance, Reliability, Collaboration, and Skills.
+
+- [BSSw Fellowship Program](https://bssw.io/fellowship)
+- [APPLY NOW!](https://bssw.io/pages/apply-for-the-bssw-fellowship-program)
+- <mark>Application deadline: Friday, October 31, 2025</mark>; this is a firm deadline that will not be extended.
+
+### Current and past BSSw Fellows are:
+
+- [Providing information on projects and perspectives (2024 BSSw Fellows)](https://bssw.io/blog_posts/2024-bssw-fellows-projects-and-perspectives)
+- [Advancing software productivity and sustainability via ongoing projects (2025 BSSw Fellows)](https://bssw.io/blog_posts/introducing-the-2025-bssw-fellows)
+- [Providing information on projects and perspectives (2023 BSSw Fellows)](https://bssw.io/blog_posts/2023-bssw-fellows-projects-and-perspectives)
+<!-- - [Advancing software productivity and sustainability via ongoing projects (2024 BSSw Fellows)](https://bssw.io/blog_posts/introducing-the-2024-bssw-fellows) -->
+- [Providing information on projects and perspectives (2022 BSSw Fellows)](https://bssw.io/blog_posts/2022-bssw-fellows-projects-and-perspectives)
+<!-- - [Advancing software productivity and sustainability via ongoing projects (2023 BSSw Fellows)](https://bssw.io/blog_posts/introducing-the-2023-bssw-fellows) -->
+- [Providing information on projects and perspectives (2021 BSSw Fellows)](https://bssw.io/blog_posts/2021-bssw-fellows-projects-and-perspectives)
+<!-- - [Providing information on projects and perspectives (2020 BSSw Fellows)](https://bssw.io/blog_posts/2020-bssw-fellows-projects-and-perspectives) -->
+<!-- - [Guiding developers through each stage of the scientific software lifecycle (2019 BSSw Fellows)](https://bssw.io/blog_posts/2019-bssw-fellows-guide-developers-through-each-stage-of-the-scientific-software-lifecycle) -->
+<!-- - [Tackling scientific productivity challenges (2018 BSSw Fellows)](https://bssw.io/blog_posts/2018-bssw-fellows-tackle-scientific-productivity-challenges) -->
```

Comment:
The HTML comments you're using in this list are not having the intended effect.  They are being treated as text, not recognized as comments, and the markdown you're trying to prevent from being rendered is actually being displayed.

I think this is a quirk of the way our markdown parser works, and the only solution I know of is to delete the commented material.